setup.py

```diff
@@ -54,6 +54,10 @@
     # Needed for packaging
     #
     "wheel",
+    #
+    # Needed for ESP-mode
+    #
+    "esptool",
```

Comment:
I think this would need to go in `mu.wheels.__init__` I thinnk

Reply:
Ok

Reply:
Done.